lib/sidekiq_unique_jobs/lua/lock.lua

```diff
@@ -62,8 +63,16 @@ if lock_type == "until_expired" and pttl and pttl > 0 then
   log_debug("ZADD", expiring_digests, current_time + pttl, digest)
   redis.call("ZADD", expiring_digests, current_time + pttl, digest)
```

Comment:
I left this alone because I don't fully understand what the expiring digests set is used for and if this optimization would be applicable to jobs utilizing an `until_expired` lock strategy.